README.md

```diff
@@ -3,8 +3,7 @@
 [![Build Status](https://travis-ci.org/commercialhaskell/stack.svg?branch=master)](https://travis-ci.org/commercialhaskell/stack)
 [![Release](https://img.shields.io/github/release/commercialhaskell/stack.svg)](https://github.com/commercialhaskell/stack/releases)
 
-`stack` is a cross-platform program for developing Haskell
-projects. It is aimed at Haskellers both new and experienced.
+`stack` is a cross-platform program for developing Haskell projects. It is aimed at Haskellers both new and experienced.
```

Comment:
Sorry, I wasn't sure it was intentional, and it's just force of habit for me to remove line breaks in anything but stuff that is only likely to be read inside a small terminal window.